tests/test_validation.py

```diff
@@ -101,7 +101,7 @@ def test_popo_validation():
 def test_raise():
     with pytest.raises(ValidationErrors) as ei:
         validate(invalid_obj, raise_exc=True)
-    assert list(ei.value)  # test iteration over errors
+    assert len(list(ei.value)) == 5  # test iteration over errors
```

Comment:
It would be better to test the actual errors in some degree rather than the magic number 5.

The comment is also no longer accurate.

You could add an explicit test for `"upload-store": 0` rather than piggyback it on here.